fatrace.c

```diff
@@ -444,7 +463,7 @@ parse_args (int argc, char** argv)
     };
 
     while (1) {
-        c = getopt_long (argc, argv, "C:co:s:tp:f:h", long_options, NULL);
+        c = getopt_long (argc, argv, "C:co:us:tp:f:h", long_options, NULL);
```

Comment:
Please keep them sorted in the same order as the long options, i.e. add the `u` after `t`.

Reply:
Done.